robot/framesystem/framesystem.go

```diff
@@ -31,16 +31,50 @@ var API = resource.APINamespaceRDKInternal.WithServiceType(SubtypeName)
 var InternalServiceName = resource.NewName(API, "builtin")
 
 // A Service that returns the frame system for a robot.
+//
+// TransformPose example:
+//
+//	// Define a Pose coincident with the world reference frame
+//	firstPose := spatialmath.NewZeroPose()
+//
+//	// Establish the world as the reference for firstPose
+//	firstPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, firstPose)
+//
+//	// Calculate firstPoseInFrame from the perspective of the origin frame of myArm
+//	transformedPoseInFrame, err := fsService.TransformPose(context.Background(), firstPoseInFrame, "myArm", nil)
+//
+// TransformPointCloud example:
+//
+//	// Create an empty slice to store point cloud data.
+//	pointClouds := make([]pointcloud.PointCloud, 0)
+//
+//	Transform the first point cloud in the list from its reference frame to the frame of 'myArm'.
+//	transformed, err := fsService.TransformPointCloud(context.Background(), pointClouds[0], referenceframe.World, "myArm")
+//
+// CurrentInputs example:
+//
+//	myCurrentInputs, err := fsService.CurrentInputs(context.Background())
+//
+// frameSystem, err := fsService.FrameSystem(context.Background(), nil)
```

Comment:
please re-format this so that is of the following form:
```
// FrameSystem example:
//
//	frameSystem, err := fsService.FrameSystem(context.Background(), nil)

```
this way we are consistent with the pattern of examples we have above.

Reply:
also please add a space between this and `type Service interface {`

Note: in case this is not possible because of how our linter works that is ok :)